bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -0,0 +1,63 @@
+package edu.gemini.spModel.target.env
+
+import edu.gemini.spModel.core.Coordinates
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.shared.util.immutable.{Option => GOption}
+import edu.gemini.shared.util.immutable.ScalaConverters._
+import edu.gemini.skycalc.{Coordinates => SCoordinates}
+
+
+import java.time.Instant
+
+import scalaz.NonEmptyList
+
+/** Collection of stars that make up the science target(s) for an observation,
+  * along with any configuration details unique to the instrument in use.
+  */
+trait Asterism {
+
+  /** All targets that comprise the asterism.  There must be at least one target.
+    */
+  def targets: NonEmptyList[SPTarget]
```

Comment:
In general I'd prefer to use the new `Target` class but I certainly see the point that the consequences downstream could be very big. Is it so that `SPTarget` has the listeners to inform about changes?